datastore/concepts.js

```diff
@@ -619,6 +619,35 @@ Metadata.prototype.testPropertyByKindRunQuery = function(callback) {
   this.datastore.runQuery(query, callback);
 };
 
+Metadata.prototype.testPropertyFilteringRunQuery = function(callback) {
+  datastore.createQuery = this.datastore.createQuery;
+
+  // [START property_filtering_run_query]
+  var query = datastore.createQuery('__property__')
+    .select('__key__')
+    .filter('__key__ >=', datastore.key(['__kind__', 'Task', '__property__', 'priority']));
```

Comment:
I would consider pulling this key out into a variable, just since it's getting pretty long.